### DISCUSSION TITLE
preview-tui issue with less version 633

### DISCUSSION THREAD

**Discussion by marcoSven** [jarun/nnn#1665] [Q&A] [answered]
I am on macOS and after upgrading to 633 from 608, https://www.greenwoodsoftware.com/less/download.html, I noticed that the directory or text file preview sometimes appear to be aligned to bottom of the window. 
I am also getting "^X" in wait message https://github.com/gwsw/less/commit/a5bdb609aa33605eee544ab4fa24524d8f41b1da. I use the default `less -P?n -R`

**Comment by luukvbaal** [comment, marked as answer]
I see the same problem (after disabling `NNN_ICONLOOKUP`).
The `-C` flag seems to help.
